GirafRest.Test/Controllers/AccountControllerTest.cs

```diff
@@ -210,7 +210,7 @@ public void Register_InputOk_Success()
                 Username = userName,
                 Password = "GenericPassword",
                 DepartmentId = DEPARTMENT_ONE,
-                Role = GirafRoles.Citizen,
+                Role = Role.Citizen,
```

Comment:
Hvorfor skal denne hedde `Role` i stedet for `GirafRole`, hvis den endelig skal være Ental?

Reply:
Under Models har vi nemlig allerede en GirafRole klasse, så vi ændrede den til Role for at undgå konflikter med den klasse.